admin/services/data-access/sql.service.js

```diff
@@ -114,6 +117,17 @@ const convertMomentToJsDate = (m) => {
   return new Date(iso)
 }
 
+/**
+ * Convert Date to Moment object
+ * Useful for converting Data during UPDATES and INSERTS
+ */
+const convertDateToMoment = (d) => {
```

Comment:
@jon-shipley Shouldn't that already be handled by date service?

Reply:
@GeorgeChatzigiannis - I think it makes more sense to keep this together, as we will also have to port this to the functions.  Last time we spun it off into its own repo.